pkg/ansible/watches/watches_test.go

```diff
@@ -19,12 +19,53 @@ import (
 	"os"
 	"path/filepath"
 	"reflect"
+	"strconv"
 	"testing"
 	"time"
 
 	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	ansibleflags "github.com/operator-framework/operator-sdk/pkg/ansible/flags"
 )
 
+func TestNew(t *testing.T) {
+	expectedGVK := schema.GroupVersionKind{
+		Version: "v1alpha1",
+		Group:   "app.example.com",
+		Kind:    "Example",
+	}
+	expectedReconcilePeriod, _ := time.ParseDuration(reconcilePeriodDefault)
+
+	watch := New(expectedGVK)
+	if watch.GroupVersionKind != expectedGVK {
+		t.Fatalf("Unexpected GVK %v expected %v", watch.GroupVersionKind, expectedGVK)
+	}
+	if watch.MaxRunnerArtifacts != maxRunnerArtifactsDefault {
+		t.Fatalf("Unexpected maxRunnerArtifacts %v expected %v", watch.MaxRunnerArtifacts, maxRunnerArtifactsDefault)
+	}
+	if watch.MaxWorkers != maxWorkersDefault {
+		t.Fatalf("Unexpected maxWorkers %v expected %v", watch.MaxWorkers, maxWorkersDefault)
+	}
+	if watch.ReconcilePeriod != expectedReconcilePeriod {
+		t.Fatalf("Unexpected reconcilePeriod %v expected %v", watch.ReconcilePeriod, expectedReconcilePeriod)
+	}
+	if watch.ManageStatus != manageStatusDefault {
+		t.Fatalf("Unexpected manageStatus %v expected %v", watch.ManageStatus, manageStatusDefault)
+	}
+	if watch.WatchDependentResources != watchDependentResourcesDefault {
+		t.Fatalf("Unexpected watchDependentResources %v expected %v", watch.WatchDependentResources, watchDependentResourcesDefault)
+	}
+	if watch.WatchClusterScopedResources != watchClusterScopedResourcesDefault {
+		t.Fatalf("Unexpected watchClusterScopedResources %v expected %v", watch.WatchClusterScopedResources, watchClusterScopedResourcesDefault)
+	}
+
+	// this should fail validate
+	err := watch.Validate()
```

Comment:
This is odd to me. We have a `New` function above which usually returns usable and valid structs. I would expect the following to be true:
```golang
watch := New(gvk)
err := watch.Validate()
// err should always be nil on a validate after calling New
if err != nil {
     ...
}
```

Reply:
If `New()` is changed to return a valid watch, the testcase(s) here will need to reflect that. It just seemed reasonable as it was to verify that a `watch` without a role or playbook would fail `Validate()`